code/modules/events/blob.dm

```diff
@@ -23,6 +23,9 @@
 		return NOT_ENOUGH_PLAYERS
 	var/mob/dead/observer/new_blob = pick(candidates)
 	var/mob/camera/blob/BC = new_blob.become_overmind()
+	if(LAZYLEN(GLOB.blobstart))
+		var/obj/effect/landmark/BLM = pick(GLOB.blobstart)

```

Comment:
You should pick this before L25 and pass through the loc through to 
https://github.com/tgstation/tgstation/blob/85ae3d7abfae721396d879d7ff2175af447eb4df/code/modules/mob/transform_procs.dm#L464